.gazelcfg.json

```diff
@@ -0,0 +1,6 @@
+{
+	"GoPrefix": "k8s.io/test-infra",
+	"SkippedPaths": [
+		"jenkins/aws-janitor"
```

Comment:
TBH, it's much easier for small things just to build it, test it and throw the result in a Docker container so that the code is effectively vendored. If vendor management didn't suck, I'd reconsider it. :)